zk/src/archive/web/js/zk/au.js

```diff
@@ -1535,6 +1539,19 @@ zAu.cmd0 = /*prototype*/ { //no uuid at all
 					zk._focusByClearBusy = false;
 				}
 			}
+			jq('html').off('keydown', zk.$void);
+		}
+	},
+	/** Disable specified component keydown while showBusy(Component, String) is called.
+	 * while component receive shiftKey and tabKey, focus previous input element.
+	 * while component receive tabKey, focus next input element.
+	 */
+	disableKeydown: function (e) {
+		e.preventDefault();
+		if (e.which == 9 && e.shiftKey) { //tabKey and shiftKey
+			jq(this).prev(':input, [href], [tabindex]:not([tabindex="-1"])').focus();
+		} else if (e.which == 9) { //tabKey
+			jq(this).next(':input, [href], [tabindex]:not([tabindex="-1"])').focus();
```

Comment:
jq(this).prevAll('button, [href], input, select, textarea,[tabindex]').filter(':not([tabindex="-1"])').first()